### ISSUE TITLE
Include reasoning why certain ciphers should be removed

### ISSUE THREAD

**Issue by claudijd** [mozilla/ssh_scan_api#80]
Source: https://twitter.com/fugueish/status/876891820134813696

Currently, we suggest removal for cases where a cipher is not in the policy.  Perhaps we need to be more specific about why a cipher is not part of a given policy.

**Comment by floatingatoll** [comment]
"Frequently requested changes", if that's a thing, would help to provide a
structure within which to refuse them.

On Mon, Jun 19, 2017 at 1:06 PM, Jonathan Claudius <notifications@github.com

**Comment by claudijd** [comment]
We should also revisit the entire Modern policy and provide justifications for each and maybe even get some community vetting of the list.  This could be a good discussion point for next week in SFO over beverages_of_choice.

**Comment by floatingatoll** [comment]
Don't block this on that, but Modern's already been under review for some time now in a bug open on their side.

**Comment by gdestuynder** [comment]
Yeah i think we have a similar "issue" with TLS guidelines, where we basically make an opinionated choice on what list you need to be following the standard we set.
It can be improved for expectations setting though, which can be by including rationales in the output of recommendations
